backend/.babelrc

```diff
@@ -0,0 +1 @@
+{  "presets": ["@babel/preset-env"]}
```

Comment:
add new line at the end of every file please

Reply:
> add new line at the end of every file please

okay